src/foraging_gui/Dialogs.py

```diff
@@ -87,21 +87,21 @@ def __init__(self, MainWindow, parent=None):
         
         self._connectSignalsSlots()
         self.MainWindow=MainWindow
-        self._Laser_1()
-        self._Laser_2()
-        self._Laser_3()
-        self._Laser_4()
+        self._LaserColor_1()
```

Comment:
Why do we have 1 - 4 here but there are only laser_1 and laser_2 in your PR description?

Reply:
They represent different things. 

LaserColor is the field highlighted below.
![image](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/assets/109394934/1f094cef-e3a4-4523-b6f2-6b4dae0e5aec)

Laser_1 and Laser_2 correspond to the lasers we actually control.

Reply:
So apparently we need better names again.

Reply:
I may use different names for `Laser=` in the GUI to avoid confusion.